src/lib/storage/lqp_view.hpp

```diff
@@ -9,12 +9,12 @@ namespace opossum {
 
 class AbstractLQPNode;
 
-class View {
+class LQPView {
```

Comment:
Mmmh, could this be understood as "view on an LQP" as opposed to "logical view on a table, defined by an LQP"?

Reply:
I wouldn't know what to expect from a class named "LogicalView", tbh. "LQPView" at least describes the data it encapsulates.
Added a comment

```
/**
 * A SQL View represented by an LQP.
 * When used in an SQL query, the SQLTranslator hooks a copy of this LQP into the LQP it creates
 */
```